makefile.in

```diff
@@ -185,19 +185,19 @@ quickclean-gdb                :
 	rm -rf build-gdb make-gdb install-gdb
 
 install-sys                   :
-	$(MKDIR_P) "$(sysroot)/usr"
-	$(MKDIR_P) "$(sysroot)/usr/bin"
+	$(MKDIR_P) "$(DESTDIR)$(sysroot)/usr"
+	$(MKDIR_P) "$(DESTDIR)$(sysroot)/usr/bin"
 	for f in "$(bindir)/"*; do \
 		if test -f "$$f"; then \
 			f="$${f##*/}"; \
-			(cd "$(sysroot)/usr/bin" && $(LN_S) -f "$(bindir)/$$f" "$${f##$(target)-}"); \
+			(cd "$(DESTDIR)/$(sysroot)/usr/bin" && $(LN_S) -f "$(bindir)/$$f" "$${f#$(target)-}";) \
```

Comment:
I suggest changing all instances of `$(DESTDIR)/...` to `$(DESTDIR)...` for consistency. The extra slash is redundant.

Reply:
Done.